plugins/outputs/yandex_cloud_monitoring/yandex_cloud_monitoring.go

```diff
@@ -0,0 +1,276 @@
+package yandex_cloud_monitoring
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/selfstat"
+)
+
+// YandexCloudMonitoring allows publishing of metrics to the Yandex Cloud Monitoring custom metrics
+// service
+type YandexCloudMonitoring struct {
+	Timeout            internal.Duration
```

Comment:
Ideally all the fields settable from toml should have the toml tags defined.

Reply:
Done